lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -38,7 +38,7 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
     ModalService.open LogoPhotoForm, group: => @group
 
   @handleSubscriptionSuccess = ->
-    if AppConfig.chargify and $location.search().chargify_success?
+    if (AppConfig.chargify or AppConfig.environment != 'production') and $location.search().chargify_success?
```

Comment:
I just wanted an easy way for mhjb to test his css / html changes. I guess
he can put chargify stuff into his env, but that seems like more hassle
than just making it easier to trigger.

On Wed, 11 Nov 2015 at 19:03 James Kiesel <notifications@github.com> wrote:

> In lineman/app/components/group_page/group_page_controller.coffee
> <https://github.com/loomio/loomio/pull/2702#discussion_r44504159>:
>
> > @@ -38,7 +38,7 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
> >      ModalService.open LogoPhotoForm, group: => @group
> >
> >    @handleSubscriptionSuccess = ->
> > -    if AppConfig.chargify and $location.search().chargify_success?
> > +    if (AppConfig.chargify or AppConfig.environment != 'production') and $location.search().chargify_success?
>
> Not sure I understand? For testing in dev we could set .env to contain
> the chargify test creds, for automated testing we could stub out AppConfig.chargify
> = true?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2702/files#r44504159>.
>